frontend/prisma/migrations/migration_lock.toml

```
@@ -1,3 +1,3 @@
 # Please do not edit this file manually
 # It should be added in your version-control system (i.e. Git)
-provider = "cockroachdb"
+provider = "cockroachdb"
```

Asi bude řvát Prettier:

```suggestion
provider = "cockroachdb"

```